recipe/build_base.sh

```diff
@@ -508,7 +508,8 @@ fi
 rm -rf ${PREFIX}/lib/python${VER}/distutils/command/*.exe
 
 python -c "import compileall,os;compileall.compile_dir(os.environ['PREFIX'])"
-rm ${PREFIX}/lib/libpython${VER}.a
+rm ${PREFIX}/lib/libpython${VERABI}.a
+
 if [[ "$target_platform" == linux-* ]]; then
```

Comment:
The `bin` directory has two executables of the same size. How about making one a symlink of the other?
```
if [[ ${PY_INTERP_DEBUG} == yes ]]; then
  rm ${PREFIX}/bin/python{VER}
  ln -s ${PREFIX}/bin/python{VERABI} ${PREFIX}/bin/python{VER}
fi
```

Reply:
Yes, I checked the artifact and the files are indeed identical, so this is probably fine.